### PR TITLE
feat(cts): add ingestion tests

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

🎟 JIRA Ticket: [DI-921](https://algolia.atlassian.net/browse/DI-921)

Add the `x-algolia-agent` query param for the CTS to work, don't merge before we allow it on the API !

Add a basic CTS for all ingestion endpoints, this is to be improved with all the oneOf.

[DI-921]: https://algolia.atlassian.net/browse/DI-921?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by millotp** [comment]
`git merge -Xours main` seems to work, only the commits here make no sense

File: generators/src/main/java/com/algolia/codegen/cts/AlgoliaCTSGenerator.java [hunk 1]
```diff
@@ -128,7 +128,6 @@ public Map<String, Object> postProcessSupportingFileData(Map<String, Object> obj
             System.out.println(e.getMessage());
             continue;
           }
-          System.err.println("\n\n");
```
Comment (shortcuts) [inline generators/src/main/java/com/algolia/codegen/cts/AlgoliaCTSGenerator.java:131]: it's not working? :(

File: tests/CTS/methods/requests/ingestion/enableTask.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "testName": "disableTask",
```
Comment (shortcuts) [inline tests/CTS/methods/requests/ingestion/enableTask.json:3]:
```suggestion
    "testName": "enableTask",
```

File: tests/CTS/methods/requests/ingestion/getRuns.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "testName": "getRun",
```
Comment (shortcuts) [inline tests/CTS/methods/requests/ingestion/getRuns.json:3]:
```suggestion
    "testName": "getRuns",
```

**Review by shortcuts — COMMENTED** [review]
w _ooooooo_ w

File: generators/src/main/java/com/algolia/codegen/cts/AlgoliaCTSGenerator.java (hunk 1 above)
Comment (millotp) [reply generators/src/main/java/com/algolia/codegen/cts/AlgoliaCTSGenerator.java:131]: ah bah `git merge -Xours main` is dangerous ahah, I didn't notice it was gone

Comment (shortcuts) [reply generators/src/main/java/com/algolia/codegen/cts/AlgoliaCTSGenerator.java]: nothing beats a `git rebase --onto main HEAD~1`

Comment (millotp) [reply generators/src/main/java/com/algolia/codegen/cts/AlgoliaCTSGenerator.java]: yes but it requires force push unfortunately, stacked PR are broken

**Review by shortcuts — APPROVED** [review]
awesome, great work!

**Review by aallam — APPROVED** [review]
🚀

**Pull request merged**